Source/SA/RFmx Instr/RFmxInstr.cs

```diff
@@ -0,0 +1,69 @@
+﻿using NationalInstruments.RFmx.InstrMX;
+
+
+namespace NationalInstruments.ReferenceDesignLibraries.SA
+{
+    public static class RFmxInstr
+    {
+        #region Type Definitions
+        public struct LoConfiguration
+        {
+            public string LoSharingMode; //{"None","Automatic","Manual"}

```

Comment:
I took the string because I wanted to allow the user to only set the mode once and apply to SG and SA, let me see if I find a more elegant way

Reply:
Oh, that makes sense. I think that this might actually be something we want to discuss further. I am wondering if a shared common type library might be a useful addition to this project. i.e. all of the common config functions should be able to share the same _CommonConfiguration_ struct because they do the same thing, only with different underlying implementations. It could be that we have a shared "LO sharing" type that can be passed to both. Thoughts?

Reply:
Yes, that is probably what we are looking for. Should I create such a library immediately to resolve this request or how should we proceed.  Proposal is: to directly add a Common proj and LV library to the solution/ LVproject